packages/ui/src/components/va-form/VaFormNew.stories.ts

```diff
@@ -1,12 +0,0 @@
-import { defineComponent } from 'vue'
```

Comment:
We need to bring back these stories fully. Right now they won't be visible in storybook.

Reply:
We also need to polish these stories and move them to storybook. (you can move to different PR or leave an issue, but there should be a follow up - we can't mark form stories as reworked)